README.md

```diff
@@ -39,6 +53,38 @@ Our last security audit was on July 31, 2015 by NCC ([results](docs/resources/nc
 
 Any security vulnerabilities can be reported to security@docker.com.
 
+### Comparative Analysis to other signing systems
+
+#### Features 
+
+Signing Mechanism | Provenance | Delegation | Key Rotation | Freshness | Thresholds
+--- | --- | --- | --- | --- | ---
+TUF | ✔ | ✔ | ✔ | ✔ | ✔  
+Notary | ✔ | ✔ | ✔ | ✔ | Partial
+Traditional Package Signing | ✔ | ❌  | ❌  | ❌  | ❌  
+rkt trust | ✔ | ❌  | ❌  | ❌  | ❌  
+Red Hat Container Image Signing | ✔ | ❌  | ❌  | ❌  | ❌  
+YUM | ✔ | ❌  | ❌  | ❌  | ❌  
+APT | ✔ | ❌  | ❌  | ❌  | ❌  
+PyPI | ✔ | ❌  | ❌  | ❌  | ❌  
+GPG | ✔ | ❌  | ❌  | ❌  | ❌  
+No Signing | Partial (TLS) | ❌  | ❌  | ❌  | ❌  
+
+#### Protection from attacks
```

Comment:
This could use a section with a one-sentence description of each attack.

Reply:
Done!